content/dstu2/condition.md

```diff
@@ -166,5 +192,5 @@ Note: abatementDateTime is not supported for diagnosis
 [category]: http://hl7.org/fhir/DSTU2/valueset-condition-category.html
 [clinical status]: http://hl7.org/fhir/DSTU2/valueset-condition-clinical.html
 [condition-category]: http://hl7.org/fhir/condition-category
-[argonaut extension-codes]: http://argonaut.hl7.org/extension-codes
+[argonaut]: http://argonaut.hl7.org
```

Comment:
There's no expectation that systems are actually URLs. For instance the system for Patient.communication.language is just "urn:ietf:bcp:47".  We should probably be linking to the valueset definitions rather than the systems. If we need to call out the systems specifically it would probably be better for them to be unlinked free text.

Linking to nothing doesn't provide any benefit. It just looks broken.